tests/test_flows/test_sklearn.py

```diff
@@ -555,3 +556,32 @@ def test_illegal_parameter_names_featureunion(self):
             ('OneHotEncoder', sklearn.preprocessing.OneHotEncoder(sparse=False, handle_unknown='ignore'))
         ]
         self.assertRaises(ValueError, sklearn.pipeline.FeatureUnion, transformer_list=transformer_list)
+
+    def test_paralizable_check(self):
+        singlecore_bagging = sklearn.ensemble.BaggingClassifier()
+        multicore_bagging = sklearn.ensemble.BaggingClassifier(n_jobs=5)
+        illegal_param_dist = {"base__n_jobs": [-1, 0, 1] }
+        legal_param_dist = {"base__max_depth": [2, 3, 4]}
+
+        legal_models = [
+            sklearn.ensemble.RandomForestClassifier(),
+            sklearn.ensemble.RandomForestClassifier(n_jobs=5),
```

Comment:
Yes, this is exactly what happens, and this is specified in the answers array. Legal means here 'will not raise an error'. i added comments to make this more clear